mrp_repair_customer_lot/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp import fields, models, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.one
```

Comment:
A partir de ahora el @api.one dejaria de usarse, con lo cual tendrías que poner @api.multi y hacer un for para cada caso.